### DISCUSSION TITLE
✅ Alt key combinations?

### DISCUSSION THREAD

**Discussion by matteck** [isontheline/pro.webssh.net#345] [Q&A] [answered]
Is it possible to use alt-key combinations e.g. alt-arrow-up, both with the on-screen and external keyboard? Thanks

**Comment by isontheline** [comment]
Hello @matteck 👋

I don't know if iOS let us to catch "alt-key". Perhaps [combo keys](https://webssh.net/documentation/help/SSH/keyboard-shortcuts-combo-keys/) will be used.

Could you tell me the use case of "alt-arrow-X"? 
I would like to reproduce on my side.

Thanks.

**Reply by matteck** [reply]
Hi, thanks for the reply

My question was badly worded. What I meant was to send the Meta or Esc+ combination with an arrow key, for example as used frequently in Fish shell.

I'm coming to WebSSH from Prompt which provides a keyboard shortcut for the Meta key, and when using an external keyboard maps Option to Meta/Esc+

I couldn't find a way of using Meta in WebSSH

Thanks

**Reply by isontheline** [reply]
Hi 👋

Thank you a lot for your feedback !

I'm creating an issue in order to track the implementation : https://github.com/isontheline/pro.webssh.net/issues/350

Have a nice day ☀️

**Reply by isontheline** [reply]
Hello @matteck 👋

WebSSH 16.2 (Build 830) now contains "Alt" virtual key (long press on "Esc" virtual key) : 
https://testflight.apple.com/join/QSrBK59z

Physical support of "Alt / Option / Meta" key will be available inside Build 831, available in a few hours.

Have a nice day ☀️

**Reply by isontheline** [reply]
@matteck Could you try WebSSH 16.2 (build 832) with physical support of Alt / Meta?
https://testflight.apple.com/join/QSrBK59z

**Comment by isontheline** [comment, marked as answer]
WebSSH 16.2 with Alt / Meta support will be released in a few days.
